python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/vxlan_interface.schema.yml

```diff
@@ -100,6 +102,16 @@ keys:
                       items:
                         type: str
                         description: Remote VTEP IP Address.
+              vlan_range:
+                type: dict
+                description: Set VNI on range of vlans.
+                keys:
+                  vlans:
+                    type: str
+                    description: e.g. 100-110
+                  vnis:
+                    type: str
+                    description: e.g. 10100-10110.
```

Comment:
Please mention number of `vnis` should be equal to number of `vlans`

Reply:
Updated `Set `vnis` in the correct sequence for the `vlans`.`